pyop2/global_kernel.py

```diff
@@ -293,7 +293,8 @@ def __init__(self, local_kernel, arguments, *,
                  constant_layers=False,
                  subset=False,
                  iteration_region=None,
-                 pass_layer_arg=False):
+                 pass_layer_arg=False,
+                 form_signature=None):
```

Comment:
```suggestion
                 signature=None):
```
We try to avoid finite-element specific things at OP2 level.